### PR TITLE
Tobikuhlmann/correct adjustment factor

### PR DESCRIPTION
**Pull request by tobikuhlmann**
This PR corrects the proposed value of the staking reward rate adjustment factor. The math to determine the parameter value has been incorrect and is corrected here. 

Math details can be found here: https://www.notion.so/clabsco/Parameter-value-choice-Dynamic-target-voting-yield-adjustment-factor-147ef4f44c894bf797ac59d736be021a

### CODE REVIEW

**Comment by tobikuhlmann** [comment]
Do not merge yet

**Comment by tobikuhlmann** [comment]
Deployed and tested on staging, adjustment works as expected now. :)
https://www.notion.so/clabsco/CGP-Testing-Documentation-Epoch-Reward-parameter-changes-f45065e950e94541b1c578d999596d5f

**Comment by tobikuhlmann** [comment]
Tested for 249 epochs on staging testnet, parameter is correct now and adjustment mechanism works as intended, ready to merge

**Review by sudarics — APPROVED** [review]
Looks good to me.

File: CGPs/0033.md [hunk 1]
```diff
@@ -42,8 +46,8 @@ https://explorer.celo.org/address/0x07F007d389883622Ef8D4d347b3f78007f28d8b7/rea
 
 
 ## Risks
-- Voting participation could be too sensitive to target voting yield changes. The value of adjustmentFactor=1/1825 could be too large in that case. However, the opposite has been the case so far: Voting yield has decreased from 0.06 to 0.053 and voting CELO fraction has been increasing from ~0.5 at epoch 50 to ~0.6 at epoch 400
-- Voting participation might not be sensitive enough. The value of adjustmentFactor=1/1825 could be too small in that case to effectively increase voting participation when it’s dropping. In that case, however, a small adjustmentFactor would be better than none at all, as it is currently.
+- Voting participation could be too sensitive to target voting yield changes. The value of 'adjustmentFactor' could be too large in that case. However, the opposite has been the case so far: Voting yield has decreased from 0.06 to 0.053 and voting CELO fraction has been increasing from ~0.5 at epoch 50 to ~0.6 at epoch 400
```
Comment (YazzyYaz) [inline CGPs/0033.md:49]: use backticks for quoting code functions
`adjustmentFactor`

File: CGPs/0033.md [hunk 2]
```diff
@@ -42,8 +46,8 @@ https://explorer.celo.org/address/0x07F007d389883622Ef8D4d347b3f78007f28d8b7/rea
...
+- Voting participation might not be sensitive enough. The value of 'adjustmentFactor' could be too small in that case to effectively increase voting participation when it’s dropping. In that case, however, a small adjustmentFactor would be better than none at all, as it is currently.
```
Comment (YazzyYaz) [inline CGPs/0033.md:50]: Use backticks here as well: ``

File: CGPs/0033.md (hunk 1 above)
Comment (tobikuhlmann) [reply CGPs/0033.md]: thanks, resolved

File: CGPs/0033.md (hunk 2 above)
Comment (tobikuhlmann) [reply CGPs/0033.md]: resolved as well

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**